components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -327,6 +327,13 @@
                     "description": "the hard limit acts as a ceiling for the soft limit. For more details please check https://man7.org/linux/man-pages/man2/getrlimit.2.html"
                 }
             }
+        },
+        "env": {
```

Comment:
How do we ship the schema, actually? Asking because I'm trying to understand how we roll out this change.
And the editor in the preview still shows the env as "not valid" (red squirrels).

Reply:
This change will apply once we deploy the changes to PAYG. The pointer to that is [here](https://github.com/SchemaStore/schemastore/blob/39733e13f775f8049a30148e79c27dbfda82324b/src/api/json/catalog.json#L2372-L2377).

Reply:
Thank you!